scripts/build/build/targets.py

```diff
@@ -253,6 +251,10 @@ def HostTargets():
         app_targets.append(target.Extend(
             'ota-requestor', app=HostApp.OTA_REQUESTOR, enable_ble=False))
         app_targets.append(target.Extend('python-bindings', app=HostApp.PYTHON_BINDINGS))
+        app_targets.append(target.Extend('tv-app', app=HostApp.TV_APP))
```

Comment:
Other targets do not use the `-app` suffix as that seems redundant (notice lock/shell/etc).

Please name just `tv/lighting/tv-casting/bridge`

Reply:
Done